### ISSUE TITLE
Translation in locale differs from entry in unofficial names (alpha2 US, locale de)

### ISSUE THREAD

**Issue by RST-J** [hexorx/countries#444]
Hello and first thanks for this incredible gem - it reduced my effort to support international addresses from close to infinite to next to nothing.
Rather by accident I found out that the translation for the US in the German [locale file](https://github.com/hexorx/countries/blob/master/lib/countries/cache/locales/de.json) is 'Vereinigte Staaten' while the corresponding German entry in the unofficial names array in [US.yml](https://github.com/hexorx/countries/blob/master/lib/countries/data/countries/US.yaml) is 'Vereinigte Staaten von Amerika'

'Vereinigte Staaten' is rather informal (probably most/all people get that) but 'Vereinigte Staaten von Amerika' is the (more) official German name, if they are not referred to as 'USA'.

Long story short: Would you accept a PR which changes the locale value? I'd also check 'inconsistencies' for all other countries in the German locale as well.
I could of course make some workarounds to get what I want, but I'd give back a little if you like.

**Comment by rposborne** [comment]
@RST-J We are always open to data corrections.  Though our translations are not currently stored in this repo, but stored in the upstream gem i18n_data.
